aries-site/src/examples/templates/dashboards/content/chartCards/CostByMonth.js

```diff
@@ -133,6 +124,7 @@ export const CostByMonth = ({ period }) => {
   };
 
   return (
+    // <Box>
```

Comment:
Remove?

Reply:
Done. Good catch. I was playing with some of the responsive aspects when card contents shift at different sizes, should cards fill their row height for a nice grid appearance or should they fit their contents. I'll pair up with @bahriaditi  when we have a few more cards so that we can work through some of those aspects.